### PR TITLE
feat : typography & lists

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR I've added typography and lists

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Understandable and clean code, good to merge

File: Components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    font-size: 1.8rem;
+}
+
+small{
+    font-size: 1.1rem;
+    font-weight: 700;
+}
+
+.text-gray{
+    color: var(--grey) !important ;
```
Comment (Megha-Joshi) [inline Components/typography/typography.css]: avoid using !important

Comment (sudipkundu999) [reply Components/typography/typography.css]: I forgot to remove it. Removing this. Thank you for pointing out.

**Review by Megha-Joshi — APPROVED** [review]
Good to merge.

**Pull request merged**